docs/authorization/roles.md

```diff
@@ -79,69 +79,93 @@ These privileges are common to both Self-Hosted DataHub and Managed DataHub.
+| Edit Owners                        | :heavy_check_mark: | :x:                | :x:                | The ability to add and remove owners of an entity.                                               |
+| Edit Group Members                 | :heavy_check_mark: | :x:                | :x:                | The ability to add and remove members to a group.                                                |
+| Edit User Profile                  | :heavy_check_mark: | :x:                | :x:                | The ability to change the user's profile including display name, bio, title, profile image, etc. |
+| Edit Contact Information           | :heavy_check_mark: | :x:                | :x:                | The ability to change the contact information such as email & chat handles.                      |
+| Delete                             | :heavy_check_mark: | :x:                | :x:                | The ability to delete the delete this entity.                                                    |
 
 #### Managed DataHub
 
 These privileges are only relevant to Managed DataHub.
 
 ##### Platform Privileges
 
-| Privilege               | Admin              | Editor             | Reader |
-|-------------------------|--------------------|--------------------|--------|
-| Create Constraints      | :heavy_check_mark: | :heavy_check_mark: | :x:    |
-| View Metadata Proposals | :heavy_check_mark: | :heavy_check_mark: | :x:    |
-| Manage Tests            | :heavy_check_mark: | :x:                | :x:    |
-| Manage Global Settings  | :heavy_check_mark: | :x:                | :x:    |
+| Privilege                   | Admin              | Editor             | Reader | Description                                                                                         |
+|-----------------------------|--------------------|--------------------|--------|-----------------------------------------------------------------------------------------------------|
+| Manage Tests                | :heavy_check_mark: | :heavy_check_mark: | :x:    | Create and remove Asset Tests.                                                                      |
+| View Metadata Proposals     | :heavy_check_mark: | :heavy_check_mark: | :x:    | View the requests tab for viewing metadata proposals.                                               |
+| Create metadata constraints | :heavy_check_mark: | :heavy_check_mark: | :x:    | Create metadata constraints.                                                                        |
+| Manage Platform Settings    | :heavy_check_mark: | :x:                | :x:    | View and change platform-level settings, like integrations & notifications.                         |
+| Manage Connections          | :heavy_check_mark: | :x:                | :x:    | Manage connections to external platforms.                                                           |
```

Comment:
How about "Manage connections to external DataHub platforms." so it's more clear that you can't connect to like snowflake or other platforms via this connection?

Reply:
Addressed.